pkg/network/usm/ebpf_javatls.go

```diff
@@ -146,6 +149,17 @@ func (p *JavaTLSProgram) GetAllUndefinedProbes() []manager.ProbeIdentificationPa
 	return []manager.ProbeIdentificationPair{{EBPFFuncName: "kprobe__do_vfs_ioctl"}}
 }
 
+// isJavaProcess checks if the given PID comm's name is java.
+// The method is much faster and efficient that using process.NewProcess(pid).Name().
+func isJavaProcess(pid int) bool {
+	content, err := os.ReadFile(filepath.Join(util.GetProcRoot(), strconv.Itoa(pid), "comm"))
```

Comment:
we should have a small waiting periode here if /proc has not been populated yet (netlink is faster than /proc)
```
	proc, err := process.NewProcess(int32(pid))
	if err != nil {
		// We receive the Exec event first and /proc could be slow to update
		end := time.Now().Add(10 * time.Millisecond)
		for end.After(time.Now()) {
			proc, err = process.NewProcess(int32(pid))
			if err == nil {
				break
			}
			time.Sleep(time.Millisecond)
		}
	}
	if err != nil {
		// short living process can hit here (or later proc.Name() parsing)
		// as they already exited when we try to find them in /proc
		// so let's be quiet on the logs as there not much to do here
		return
	}
```